MAUI/Autocomplete/UI-Customization.md

```diff
@@ -1212,3 +1212,88 @@ N> Dynamic changes to the `HorizontalTextAlignment` property may not be function
 The following image illustrates the result of the above code:
 
 ![TextAlignment](Images/UICustomization/TextAlignment.png)
+
+## Clear button customization
+
+The `ClearButtonPath` property is used to customize the appearance of the clear button.
```

Comment:
This is so simple, In UG we have to provide adequate info.

Reply:
change updated